src/main/java/net/snowflake/client/core/EventHandler.java

```diff
@@ -176,7 +176,7 @@ public synchronized long getLogBufferSize() {
 
   /** Creates and runs a new QueueFlusher thread */
   synchronized void startFlusher() {
-    // Create a new scheduled executor service with a threadfactory that
+    // Create a new scheduled executor service with a thread factory that
```

Comment:
I didn't change all of the words like this, but there really didn't seem to be any value in this being merged together